osquery/registry/registry_factory.cpp

```diff
@@ -224,6 +225,10 @@ std::string RegistryFactory::getActive(const std::string& registry_name) const {
 }
 
 void RegistryFactory::setUp() {
+  if (shutdownRequested()) {
```

Comment:
Does the check at https://github.com/osquery/osquery/pull/7106/files#diff-691b47a21b0ad765c31fe03b547baf942a06cd3c6ce5770a11997bec3c7d9977R567 immediately proceed this one?